contracts/Vault.sol

```diff
@@ -39,6 +47,10 @@ contract Vault is ERC4626Metadata, Ownable {
         return _decimals;
     }
 
+    function getLockDuration() public view returns (uint256) {
+        return _lockDuration;
```

Comment:
why this, instead of making lockDuration public?

Reply:
I don't mind this pattern if you have the function as an interface function.